src/components/components/member/member.js

```diff
@@ -540,19 +540,21 @@ class Member extends React.PureComponent {
     );
 
     return (
-      <LayoutContained className="member">
```

Comment:
@alexdor this member class is missing on the dynamic generated member pages for the production environment and this is the reason the layout breaks (it is undefined..). I cannot reproduce this issue in my development environment neither on development, nor on production build and this is the reason I added another div to bypass the issue, hoping this will work. If you have a better idea it is welcome!